### ISSUE TITLE
The difference between trainning from image-classification task pretrained model and training from the start 

### ISSUE THREAD

**Issue by jczhydy** [pavancm/CONTRIQUE#15]
When training, I find training with pretrained resnet50 model takes less trainning time.I want to know why the article choose to training from the start without pretrained?

**Comment by pavancm** [comment]
Pretrained resnet50 is a supervised trained model which requires object classification labels as ground truth for training. There is a limitation on dataset size based on how easy it is to obtain accurate ground truth labels for object classes. On the other hand this work focuses on self-supervision where ground truth labels can be generated without manual supervision, thus making it easy to create large datasets. We also showed that these self-supervised models achieve competitive performance. The objective was to show that supervised data is not essential for achieving superior performance.

**Comment by jczhydy** [comment]
Thank you.
  And I want to learn more about MS colorspace,I failed to get  more information when searching for it on the internet. When I train the model only through MS colorpace by the code provided in this website,I can't get a good result like other colorspace like RGB,Grayscale.
